com.unity.perception/Runtime/GroundTruth/Labelers/KeypointLabeler.cs

```diff
@@ -332,20 +332,21 @@ bool DoesTemplateContainJoint(JointLabel jointLabel)
 
         void ProcessLabel(Labeling labeledEntity)
         {
-            // Cache out the data of a labeled game object the first time we see it, this will
-            // save performance each frame. Also checks to see if a labeled game object can be annotated.
-            if (!m_KnownStatus.ContainsKey(labeledEntity.instanceId))
+            if (idLabelConfig.TryGetLabelEntryFromInstanceId(labeledEntity.instanceId, out var labelEntry))
```

Comment:
For future reference, avoid surrounding entire method bodies in 'if' statements. Instead, do `if (!xxx) return;`

Reply:
I will change this. 
Is there a C# or Unity coding conventions reference specifically on this topic? (early return vs single return)

Reply:
nothing specific afaik - it's a more subjective assessment